### PR TITLE
det frontend

### LINKED ISSUE #2692: det
Add frontend function with for TensorFlow, task listed in #1546

### PR DESCRIPTION
**Pull request by CatB1t**
fix #2692

### CODE REVIEW

File: ivy/functional/frontends/tensorflow/__init__.py [hunk 1]
```diff
@@ -1,5 +1,4 @@
 # flake8: noqa
-# flake8: noqa
```
Comment (johanjino) [inline ivy/functional/frontends/tensorflow/__init__.py:2]: Please make sure that no parts of the code which isn't related to the task is changed. It causes conflicts and errors especially while performing lint tests 😀

**Comment by johanjino** [comment]
Meanwhile, Ill keep verifying your code and let you know if anything else is required 😄

**Comment by CatB1t** [comment]
Hi @johanjino, updated the PR as required.

**Comment by johanjino** [comment]
Hi @CatB1t, looks perfect to me. Thank you for your contribution 😃

**Pull request merged**